phpstan fixes (php doc blocks)

williamdes opened the pull request.

williamdes commented:
@fredericdelordm some basic fixes

iulian03 commented on MangoPay/Libraries/ApiOAuth.php:
```diff
@@ -30,6 +30,7 @@ public function CreateToken()
             $rest->AddRequestHttpHeader('x-tenant-id: uk');
         }
         $response = $rest->Request($urlMethod, $requestType, $requestData);
+        /** @var \MangoPay\Libraries\OAuthToken $token */
```
No need for doc in the middle of the algorithm

williamdes replied:
```suggestion
```

iulian03 reviewed (commented):
Looking good. I've added a single comment.

williamdes commented:
> Looking good. I've added a single comment.

Thanks, I removed it.
That said, it was important for phpstan analysis since it can not figure out what type is the object.

williamdes commented:
Hi @iulian03 
Does this PR look okay now ?
